frontend/taipy-gui/base/src/app.ts

```diff
@@ -80,6 +93,10 @@ export class TaipyApp {
         this._onChange = handler;
```

Comment:
this._onChange = handler.bind(this)

Reply:
I tried this one but it doesnt work unfortunately. After testing I dont think that they work with classes.
```javacript
class A {
    a = undefined;
    constructor() {
        this.a = ((a) => {
            console.log(a)
        }).bind(this);
    }
}
```

Reply:
It does
But I think there's a difference between function and arrow function. 
No big deal anyway